### ISSUE TITLE
File Bundling Issue

### ISSUE THREAD

**Issue by adammahon** [Shazwazza/Smidge#68]
I started testing out Smidge today, and I ran into an issue with how css & js is bundled. It appears that when css/js files are bundled together a semicolon is added between the contents of each file (two in the case of js). This isn't much of an issue with javascript, but with css this prevents the first rule after the semicolon from being processed by the browser. 

In case it helps, below is the source I was running when I found the issue:
```js
/* fileOne.js */
console.log("file one");
```
```js
/* fileTwo.js */
console.log("file two");
```
```css
/* fileOne.css */
h1{ color: blue; }
```
```css
/* fileTwo.css */
p{ color: red; }
```
```cshtml
@{
    SmidgeHelper
        .CreateCssBundle("cssbundle")
        .RequiresCss("~/css/file-one.css", "~/css/file-two.css");

    SmidgeHelper
        .CreateJsBundle("jsbundle")
        .RequiresJs("~/js/file-one.js", "~/js/file-two.js");
}

<!DOCTYPE html>
<html>
    <head>
        <link href="cssbundle" rel="stylesheet" />
    </head>
    <body>
        <h1>Hello World</h1>
        <p>How are you today?</p>
        
        <script type="text/javascript" src="jsbundle"></script>
    </body>
</html>
```
Whats output to the browser:
```js
console.log("file-one");;;console.log("file-two");;;
```
```css
h1{color:blue;};p{color:red;};
```

**Comment by Shazwazza** [comment]
Sorry have been on vacation, I'll have a look at this real soon, thanks for reporting!

**Comment by Shazwazza** [comment]
Oops sorry this fell off the radar! Ill have a look first thing tomorrow, i have a reminder set

**Comment by Shazwazza** [comment]
I've pushed up a fix for this now, 3.0.1 is now available on Nuget.
